discord/asset.py

```diff
@@ -365,12 +365,12 @@ def with_format(self, format: ValidAssetFormatTypes, /) -> Asset:
         Parameters
         ------------
         format: :class:`str`
-            The new format of the asset.
+            The new format of an asset.
```

Comment:
```suggestion
            The new format of the asset.
```
I prefer the original here